docker/web/Dockerfile

```diff
@@ -22,5 +22,5 @@ COPY ./ /opt/ro_help
 
 WORKDIR /opt/ro_help/ro_help
 
-ENTRYPOINT ["/opt/ro_help/docker-entrypoint"]
+ENTRYPOINT ["/opt/ro_help/docker/web/docker-entrypoint"]
```

Comment:
Hmm....I'm not sure if this will work since `docker-entrypoint` will be executed relative to `web`, but it needs to be at `/opt/ro_help` level.

I saw some patterns in which the idea was to copy `./docker/entrypoint` into `/`. In this way, we can have different binaries, configs (thinking about `docker/etc/.., docker/usr/...`). After that, we just need to adapt `docker-entrypoint` to run commands with `/opt/ro_help/manage.py` instead of just `manage.py`

Reply:
agree, good point ! I'll have a look into this

Reply:
Updated with a new commit. Let me know if it is what you had in mind. Thanks !